dashboard/modules/metrics/metrics_head.py

```diff
@@ -37,6 +37,8 @@
 
 DEFAULT_PROMETHEUS_HOST = "http://localhost:9090"
 PROMETHEUS_HOST_ENV_VAR = "RAY_PROMETHEUS_HOST"
+DEFAULT_PROMETHEUS_NAME = "Prometheus"
+PROMETHEUS_NAME_ENV_VAR = "RAY_PROMETHEUS_NAME"
```

Comment:
Can you add documentation on this env var and this multi-datasource pattern in `configure-manage-dashboard.rst` doc?

Reply:
done